detection_rules/rule.py

```diff
@@ -561,6 +562,7 @@ class QueryRuleData(BaseRuleData):
     index: Optional[List[str]]
     query: str
     language: definitions.FilterLanguages
+    alert_suppression: Optional[AlertSuppressionMapping] = field(metadata=dict(metadata=dict(min_compat="8.8")))
```

Comment:
if we are min compat on 8.8 for the whole field, we can remove each of them on the subfields since they are lower (duration, missing_fields_strategy)